File: Frontend/src/app/model/patientHealthData.ts
export class PatientHealthData {
    recordId: number;
    bloodPressure: number;
    bloodSugar: number;
    heartRate: number;
    oxygenLevel: number;
    weight: number;
    height: number;
    bmi: number;
    bodyTemperature: number;
    dateTimeStats: string;
    userId: number;

    constructor(recordId: number, bloodPressure: number, bloodSugar: number, heartRate: number, oxygenLevel: number, weight: number, height: number, bmi: number, bodyTemperature: number, dateTimeStats: string, userId: number) {
        this.recordId = recordId;
        this.bloodPressure = bloodPressure;
        this.bloodSugar = bloodSugar;
        this.heartRate = heartRate;
        this.oxygenLevel = oxygenLevel;
        this.weight = weight;
        this.height = height;
        this.bmi = bmi;
        this.bodyTemperature = bodyTemperature;
        this.dateTimeStats = dateTimeStats;
        this.userId = userId;
    }
}

File: Backend/src/main/java/edu/icet/dto/PatientHealthData.java
package edu.icet.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class PatientHealthData {
    private int recordId;
    private double bloodPressure;
    private double bloodSugar;
    private double heartRate;
    private double oxygenLevel;
    private double weight;
    private double height;
    private double bmi;
    private double bodyTemperature;
    private String dateTimeStats;
    private int userId;
}
